cmd/kube-proxy/app/server.go

```diff
@@ -314,6 +315,9 @@ func (o *Options) Run() error {
 // runLoop will watch on the update change of the proxy server's configuration file.
 // Return an error when updated
 func (o *Options) runLoop() error {
+	if o.CleanupAndExit {
```

Comment:
Why not lift this into Run() above?

Reply:
Yeah, this can be checked earlier.
Thanks